test/service_instance_and_binding_test/service_instance_test/service_instance_test.go

```diff
@@ -250,6 +251,103 @@ var _ = DescribeTestsFor(TestCase{
 				ctx.CleanupAdditionalResources()
 			})
 
+			Describe("GET Service Instance Parameters, Service does not exist", func() {
+				It("Should return an error", func() {
+					ctx.SMWithOAuthForTenant.GET(web.ServiceInstancesURL + "/jkljljk/parameters").Expect().
+						Status(http.StatusNotFound).JSON().Object().Value("error").String().Equal("NotFound")
+				})
+			})
+
+			Describe("GET Service Instance Parameters,service exists", func() {
+					var instanceName string
+					var serviceID string
+					JustBeforeEach(func() {
+						Expect(serviceID).ToNot(BeEmpty())
+						planId := findPlanIDForCatalogID(ctx, brokerID, serviceID, plan1CatalogID)
+						EnsurePlanVisibility(ctx.SMRepository, TenantIdentifier, types.SMPlatform, planId, TenantIDValue)
+						postInstanceRequest["service_plan_id"] = planId
+						resp := createInstance(ctx.SMWithOAuthForTenant, "", http.StatusCreated)
```

Comment:
we create an instance also at 264, perhaps you can add a common when and reuse it in this test

Reply:
i don't see that i create an instance before